packages/neos-ui-editors/src/CodeMirror/CodeMirrorWrap/codemirror.vanilla-css

```diff
@@ -3,7 +3,7 @@
 .CodeMirror {
   /* Set height, width, borders, and global font properties here */
   font-family: monospace;
-  height: 300px;
+  height: 85vh;
```

Comment:
That would be a little bit more complicated than just a style property but I can try :1st_place_medal: